HousingRepairsSchedulingApi.Tests/UseCasesTests/BookAppointmentUseCaseTests.cs

```diff
@@ -80,6 +80,23 @@ public BookAppointmentUseCaseTests()
             await act.Should().ThrowExactlyAsync<T>();
         }
 
+        [Fact]
+#pragma warning disable xUnit1026
+#pragma warning disable CA1707
+        public async void GivenAnInvalidOrderComments_WhenExecute_ThenExceptionIsThrown()
```

Comment:
Add a test for other end of the character limit (i.e. when `orderComments` is `0`)